ui/OnboardingAskName.js

```diff
@@ -13,13 +13,16 @@ import {
 } from '../trvl'
 
 export default class OnboardingAskName extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
It's better to use placeholder prop instead of a default value (official docs start with a wrong example)
https://facebook.github.io/react-native/docs/textinput.html#placeholder

also check out this cool flag:
https://facebook.github.io/react-native/docs/textinput.html#returnkeytype